samcli/commands/sync/command.py

```diff
@@ -251,7 +264,7 @@ def do_cli(
         build_dir=build_dir,
         cache_dir=DEFAULT_CACHE_DIR,
         clean=True,
-        use_container=False,
+        use_container=use_container,
```

Comment:
@hnnasit can you confirm that individual sync flows will pick-up this value and use it for function sync flows?